pkg/controller/kogitodataindex/resource/kafka_test.go

```diff
@@ -94,7 +98,7 @@ func TestIsKafkaServerURIResolved(t *testing.T) {
 				&v1alpha1.KogitoDataIndex{},
 				cli,
 			},
-			true,
+			false,
```

Comment:
What is the difference in "AnyInstanceURI" test and "NoServer" test? Both provide empty KogitoDataIndex and expect URL not to be resolved.

Reply:
The test case "AnyInstanceURI" doesn't make any sense anymore. I'll change it.